src/Conf.ml

```diff
@@ -419,21 +429,32 @@ module Formatting = struct
        `wrap-comments = {false,true}` line."
     in
     let env = Arg.env_var "OCAMLFORMAT_WRAP_COMMENTS" in
-    C.flag ~names:["wrap-comments"] ~doc ~env ~allow_inline:true ~update:
-      (fun conf x -> {conf with wrap_comments= x} )
+    C.flag ~default:false ~names:["wrap-comments"] ~doc ~env
+      ~allow_inline:true ~update:(fun conf x -> {conf with wrap_comments= x}
+    )
+
+  let wrap_fun_args =
+    let doc =
+      "Style for function call and function definition. Can be set in a \
+       config file with an `wrap-fun-args = {true,false}` line."
+    in
+    let env = Arg.env_var "OCAMLFORMAT_WRAP_FUN_ARGS" in
+    let names = ["wrap-fun-args"] in
+    C.flag ~default:true ~names ~env ~doc ~allow_inline:true
+      ~update:(fun conf wrap_fun_args -> {conf with wrap_fun_args} )
 end
 
 (* Flags that can be modified in the config file that don't affect
    formatting *)
 
-let no_comment_check =
+let comment_check =
   let doc =
-    "UNSAFE: Do not perform any checking of comments and documentation \
-     comments."
+    "UNSAFE: Control wether to check comments and documentation comments."
```

Comment:
Maybe explicitly let-bind default before doc to try to make clear that the doc string is for the inverted flag. This gets tricky.